content/posts/glazier.md

```diff
@@ -79,16 +79,16 @@ In order to achieve this, you'll need to do the following steps on **each device
    Version          : 2.2.3267.0
    ```
 
-1. Copy the entire contents of `C:\drivers` on the laptop to `C:\drivers` on your Windows 10 virtual machine. 
+1. Copy the entire contents of `C:\drivers` on the laptop to `C:\drivers\dell\xps\9370` on your Windows 10 virtual machine. 
```

Comment:
Thanks for suggesting this! I agree it is a good idea to organize drivers [_that will be injected into the host OS from the Glazier distribution point_](https://github.com/google/glazier/blob/master/glazier/lib/actions/drivers.py#L71) this way.

However, I'm not sure it provides much value for the process of injecting drivers into WinPE from the "wim factory virtual machine"? See https://github.com/OSDeploy/OSD/blob/master/Public/OSDCloud/Edit-OSDCloud.winpe.ps1#L118 for context of what happens to these drivers. 

Does this make sense? Let me know your thoughts.

Reply:
Technically `Edit-OSDCloud.winpe` does support [specifying multiple driver paths](https://github.com/OSDeploy/OSD/blob/master/Public/OSDCloud/Edit-OSDCloud.winpe.ps1#L29), so I could write a powershell loop to search each subdirectory for the inf folders but, again, I'm not sure if doing it is worth it because: (a) the Windows 10 wim factory vm is meant to ephemeral and (b) theoretically I would store the same drivers, with a better organization structure, on my distribution point.

Reply:
The aim here is just to note to the reader that you'll probably not want to have drivers all under the same folder, but rather use model/os/etc. specific folders, whichever method works best to accomplish that sounds fine.